selectors-4/Overview.bs

```diff
@@ -2280,19 +2269,34 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 	but with different user settings,
 	can make different choices as to when an element matches '':focus-visible''.
 
-	<div class=example>
-		The following guidelines are suggested heuristics
-		for choosing when to apply '':focus-visible''
-		to elements without “native” focus indicator behavior:
-
-		* If the element received focus via a keyboard interaction,
-			including indirectly,
-			such as triggering a dialog
-			by pressing a button using the keyboard,
-			apply '':focus-visible''.
-		* If a keyboard event occurs while an element is focused,
-			even if the element wasn't focused by a keyboard interaction,
-			apply '':focus-visible''.
+	Since part of the problem is user agents' default focus indicator behaviour,
+	it is recommended that browsers implementing the `:focus-visible`
+	selector also use it in their user agent stylesheets to show the default focus
```

Comment:
I don't think "show" is quite the right verb here.  It's possible it could even be "explain", or perhaps "specify".  It might also help to add an additional clarifying sentence saying that the point is that when `:focus-visible` matches should correspond to the UA's default behavior for when to show focus indicators.  In fact, it might be worth stating *that* as a conformance requirement (and then saying that an easy way to implement that requirement may be by using `:focus-visible` in the UA style sheet.)